PCbuild/get_externals.bat

```diff
@@ -50,7 +50,7 @@ echo.Fetching external libraries...
 set libraries=
 set libraries=%libraries%                                       bzip2-1.0.6
 if NOT "%IncludeSSLSrc%"=="false" set libraries=%libraries%     openssl-1.1.0f
-set libraries=%libraries%                                       sqlite-3.14.2.0
+set libraries=%libraries%                                       sqlite-3.21.0.0
```

Comment:
@zware Have you pushed sources up here yet? I haven't.

Reply:
I have not.  @Mariatta, if you'd like to do so, the undocumented process is:

1. clone [cpython-source-deps](../../cpython-source-deps)
2. check out the `sqlite` branch
3. download the sqlite source from sqlite.org
4. unzip it into the cpython-source-deps checkout
5. commit, push (as a PR if you'd like confirmation from @zooba or me; base the PR on the `sqlite` branch)
6. tag the commit as `sqlite-<full version specified in `PCbuild/get_externals.bat`

After that, convince AppVeyor to run it again and all should be good.  This can all be done from non-Windows.

Reply:
Thanks @zware and @zooba. I made the PR in cpython-source-deps repo.
🤔 Where should the above steps be documented? Dev Guide?

Reply:
Devguide would be the right place, but *where* in the devguide I have no idea :)